### PR TITLE
volctl depends on pulsectl since v0.9.0

### PR DESCRIPTION
**Pull request by jpastuszek**
https://github.com/buzz/volctl/commit/ef7fa2386f9a793b65027f727cbd068e8a2d9167

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

I was getting this errors after update (for a while now):

```
2021-08-31_16:33:52.93948 ModuleNotFoundError: No module named 'pulsectl'
2021-08-31_16:33:54.25495 Traceback (most recent call last):
2021-08-31_16:33:54.25499   File "/bin/volctl", line 33, in <module>
2021-08-31_16:33:54.25513     sys.exit(load_entry_point('volctl==0.9.0', 'gui_scripts', 'volctl')())
2021-08-31_16:33:54.25515   File "/bin/volctl", line 25, in importlib_load_entry_point
2021-08-31_16:33:54.25526     return next(matches).load()
2021-08-31_16:33:54.25527   File "/usr/lib/python3.9/importlib/metadata.py", line 77, in load
2021-08-31_16:33:54.25540     module = import_module(match.group('module'))
2021-08-31_16:33:54.25542   File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
2021-08-31_16:33:54.25557     return _bootstrap._gcd_import(name[level:], package, level)
2021-08-31_16:33:54.25559   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
2021-08-31_16:33:54.25572   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
2021-08-31_16:33:54.25583   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
2021-08-31_16:33:54.25595   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
2021-08-31_16:33:54.25607   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
2021-08-31_16:33:54.25619   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
2021-08-31_16:33:54.25630   File "/usr/lib/python3.9/site-packages/volctl/__main__.py", line 12, in <module>
2021-08-31_16:33:54.25640     from volctl.app import VolctlApp
2021-08-31_16:33:54.25642   File "/usr/lib/python3.9/site-packages/volctl/app.py", line 19, in <module>
2021-08-31_16:33:54.25654     from volctl.pulsemgr import PulseManager
2021-08-31_16:33:54.25655   File "/usr/lib/python3.9/site-packages/volctl/pulsemgr.py", line 7, in <module>
2021-08-31_16:33:54.25664     from pulsectl import (
```

Installing `python3-pulsectl` fixes the issue.
Looks like for v0.9.0 the developer started depending on this Python package:

https://github.com/buzz/volctl/commit/ef7fa2386f9a793b65027f727cbd068e8a2d9167

### CODE REVIEW

File: srcpkgs/volctl/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'volctl'
 pkgname=volctl
 version=0.9.0
-revision=1
+revision=2
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-gobject gtk+3 desktop-file-utils python3-cairo python3-yaml python3-click"
+depends="python3-gobject gtk+3 desktop-file-utils python3-cairo python3-yaml python3-click python3-pulsectl"
```
Comment (paper42) [inline srcpkgs/volctl/template:7]: It sounds like we don't need click and yaml now, could you verify that?

**Comment by jpastuszek** [comment]
Yeah, good point, it works fine without this dependencies.

**Comment by paper42** [comment]
> Yeah, good point, it works fine without this dependencies.

Thank you for noticing this.

**Comment by paper42** [comment]
one commit for one package next time, the commits should have been squashed

**Pull request merged**